### PR TITLE
Add Tests for metastore commands using Trino and DBT

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: .github/workflows/nessie.yaml [hunk 1]
```diff
@@ -435,6 +435,88 @@ jobs:
...
+        if: ${{ failure() }}
+        continue-on-error: true
+        working-directory: test/lakectl_metastore
+        run: docker-compose logs --tail=1000 lakefs
+
+      - name: dbt Logs with gateway failure
+        if: ${{ failure() }}
+        continue-on-error: true
+        working-directory: test/lakectl_metastore
+        run: docker-compose logs --tail=1000 lakefs
```
[the remark this answers is not in the document]
Comment (guy-har) [reply .github/workflows/nessie.yaml:517]: Changed, thanks!

File: test/lakectl_metastore/etc/spark-thrift-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+MAX_ATTEMPTS=15
+
+attempt_counter=0
+while ! /opt/bitnami/spark/bin/spark-submit --class org.apache.spark.sql.hive.thriftserver.HiveThriftServer2; do
+  if [ ${attempt_counter} -eq ${MAX_ATTEMPTS} ]; then
+    echo "Operation got to max attempts"
+    exit 1
+  fi
```
[the remark this answers is not in the document]
Comment (guy-har) [reply test/lakectl_metastore/etc/spark-thrift-entrypoint.sh:10]: @nopcoder Why? we are starting from 0

File: test/lakectl_metastore/setup-test.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+#
+# setup-test.sh - set up lakeFS to run Spark app tests.  Run once per lakeFS.
+
+docker-compose exec -T lakefs /app/wait-for localhost:8000
+
+docker-compose exec -T lakefs sh -c 'lakefs setup --user-name tester --access-key-id ${LAKECTL_CREDENTIALS_ACCESS_KEY_ID} --secret-access-key ${LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY}'
+
+docker-compose exec -T lakefs sh -c 'lakectl repo create lakefs://example ${STORAGE_NAMESPACE} -d main'
+# create base schema
+ # another option would be to run as part of workflow
```
[the remark this answers is not in the document]
Comment (guy-har) [reply test/lakectl_metastore/setup-test.sh:11]: No, it's not. Rewmoved

File: test/lakectl_metastore/etc/spark-thrift-entrypoint.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (nopcoder) [reply test/lakectl_metastore/etc/spark-thrift-entrypoint.sh:10]: My bad - all good!

**Review by nopcoder — APPROVED** [review]

**Pull request merged**